.github/workflows/ci.yml

```diff
@@ -19,8 +19,7 @@ jobs:
             path: ./kernel
       - name: Install spell
         run: |
-          sudo apt-get install spell
-          sudo apt-get install util-linux
+          sudo apt install spell util-linux uncrustify
```

Comment:
This was a lazy attempt to make `uncrustify` available to the other CI steps, which didn't work. Determining the right way to do this will require more digging, which I'm not going to get around to any time soon.

Reply:
Each workflow file gets run in a separate container.